xrpl4j-core/src/test/java/org/xrpl/xrpl4j/codec/binary/types/AmountTypeTest.java

```diff
@@ -217,4 +220,31 @@ void encodeLargeNegativeCurrencyAmount() {
     assertThat(codec.fromJson(json).toHex()).isEqualTo(hex);
   }
 
+  @Test
+  void encodeDecodeMptAmount() {
+    String json = "{\"value\":\"100\",\"mpt_issuance_id\":\"00002403C84A0A28E0190E208E982C352BBD5006600555CF\"}";
+    AmountType fromJson = codec.fromJson(json);
+    assertThat(fromJson.toHex())
+      .isEqualTo("60000000000000006400002403C84A0A28E0190E208E982C352BBD5006600555CF");
+    assertThat(fromJson.toJson().toString()).isEqualTo(json);
+  }
+
+  @Test
+  void encodeDecodeLargestAmount() {
```

Comment:
We should add equivalents of these three, but for negative numbers.

Reply:
added in https://github.com/XRPLF/xrpl4j/pull/556/commits/73e282ed78fa791fd33d801ca76174b65eae42bf